app/screens/user_profile/user_profile.js

```diff
@@ -223,11 +237,11 @@ export default class UserProfile extends PureComponent {
                         <Text style={style.username}>{`@${user.username}`}</Text>
                     </View>
                     <View style={style.content}>
-                        {enableTimezone && this.buildTimezoneBlock()}
-                        {this.buildDisplayBlock('username')}
-                        {config.ShowEmailAddress === 'true' && this.buildDisplayBlock('email')}
-                        {this.buildDisplayBlock('nickname')}
-                        {this.buildDisplayBlock('position')}
+                        {!user.is_bot && enableTimezone && this.buildTimezoneBlock()}
```

Comment:
nit: we can check for `!user.is_bot` on the view level instead of the properties

Reply:
Fixed.